config/sidekiq_scheduler.yml

```diff
@@ -102,4 +102,8 @@ VBADocuments::UploadRemover:
 
 HCA::SubmissionFailureEmailAnalyticsJob:
   cron: "0 1 * * * America/New_York"
-  description: "posts HCA Submission Failure email sends and failures to Google Analytics"
\ No newline at end of file
+  description: "posts HCA Submission Failure email sends and failures to Google Analytics"
+
+DeleteOldTransactionsJob:
+  cron: "0 3 * * * America/New_york" # Daily @ 3am Eastern
```

Comment:
@rlbaker @wyattwalter Thoughts on the frequency of this job?

Reply:
Daily at 3am works for me. @wyattwalter are there any best practices we've gathered around scheduling of sidekiq jobs?

Reply:
I don't know that I'd go so far as to say it's a "best practice", but if we want some alerting around a job then it'd be ideal to not be in the middle of the night. Otherwise, whatever makes sense for the type of maintenance you're doing is fine.